commons/src/main/java/io/aiven/kafka/connect/common/source/input/ParquetTransformer.java

```diff
@@ -46,6 +48,7 @@
 
 public class ParquetTransformer implements Transformer {
```

Comment:
This is Parquet, does the AvroData class also handle parquet?

Reply:
yes, it can handle.

Reply:
Just to confirm, as a column store, Parquet _can_ supply rows/records with the Avro *data model* (unrelated to Avro serialization or binary format, unless you want it to be...)